lib/anemone/page.rb

```diff
@@ -213,5 +226,10 @@ def self.from_hash(hash)
       end
       page
     end
+
+    def no_follow?
```

Comment:
no_index might be a better name - since it's the robots no index we are checking.